src/browser/modules/D3Visualization/components/styled.tsx

```diff
@@ -170,6 +195,16 @@ export const StyledInlineList = styled.ul`
   word-break: break-word;
 `
 
+export const StyledInlineListStylePicker = styled(StyledInlineList)<{
+  frameHeight: number
+}>`
+  display: grid;
+  overflow-y: auto;
+  max-height: ${props => props.frameHeight - 75}px;
```

Comment:
Is this 75 the header height? If so is there a constant we can import for it, or if not, create one?

Reply:
There's definitely a need for more constants, haven't got a round to it yet. Looking to do so tomorrow